grammars/jinja templates.cson

```diff
@@ -96,7 +96,7 @@
         'match': '(?<=\\{\\%-|\\{\\%)\\s*\\b([a-zA-Z_][a-zA-Z0-9_]*)\\b(?!\\s*[,=])'
       }
       {
-        'match': '\\b(and|as|else|if|in|import|not|or|recursive|with(out)?\\s+context)\\b'
```

Comment:
why has the `as` keyword been removed. `{% import 'foo.html' as bar %}` is valid and should be highlighted as such.